### PR TITLE
docs: add basic example of fetch

### PR DESCRIPTION
**Pull request by o-t-w**
Adding a simple usage example of Fetch and a very brief explanation of Node Streams vs Web Streams. 

https://github.com/nodejs/undici/issues/940

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -166,6 +166,31 @@ Only supported on Node 16+.
...
+Basic usage example:
+
+```
+    import {fetch} from 'undici';
+    
+    async function fetchJson() {
+    const res = await fetch('https://example.com')
+    const json = await res.json()
+    console.log(json);
+    }
```
Comment (mcollina) [inline README.md:178]:
```suggestion
    async function fetchJson() {
        const res = await fetch('https://example.com')
        const json = await res.json()
        console.log(json);
    }
```

File: README.md [hunk 2]
```diff
@@ -166,6 +166,31 @@ Only supported on Node 16+.
...
+
+Nodejs has two kinds of streams: [web streams](https://nodejs.org/dist/latest-v16.x/docs/api/webstreams.html) which follow the API of the WHATWG web standard found in browsers, and an older Node-specific [streams API](https://nodejs.org/api/stream.html). `response.body` returns a readable web stream. If you require a Node stream, it is possible to convert between the two using `.fromWeb()`. 
+
+```
+    import {fetch} from 'undici';
+    
+    async function fetchStream() {
+    const response = await fetch('https://example.com')
+    const stream = response.body;
+    }
```
Comment (mcollina) [inline README.md:191]:
```suggestion
    async function fetchStream() {
        const response = await fetch('https://example.com')
        const stream = response.body;
    }
```

File: README.md [hunk 3]
```diff
@@ -166,6 +166,31 @@ Only supported on Node 16+.
...
+#### `response.body`
+
+Nodejs has two kinds of streams: [web streams](https://nodejs.org/dist/latest-v16.x/docs/api/webstreams.html) which follow the API of the WHATWG web standard found in browsers, and an older Node-specific [streams API](https://nodejs.org/api/stream.html). `response.body` returns a readable web stream. If you require a Node stream, it is possible to convert between the two using `.fromWeb()`. 
+
+```
+    import {fetch} from 'undici';
+    
+    async function fetchStream() {
+        const response = await fetch('https://example.com')
+        const stream = response.body;
```
Comment (ronag) [inline README.md:190]: Missing fromWeb?

Comment (o-t-w) [reply README.md:190]: I mentioned `.fromWeb()` as a brief aside. The main purpose of the code block you highlighted is just to make it super clear that response.body is a stream.

File: README.md [hunk 4]
```diff
@@ -166,6 +166,38 @@ Only supported on Node 16+.
...
+    }
+```
+
+If you require a Node stream, it is possible to convert between the two using `.fromWeb()`.
```
Comment (mcollina) [inline README.md:194]: Can you make that a link?

Comment (ronag) [reply README.md:194]: it's actually `.fromWeb()` and `.toWeb()` if you are referring to converting between the two, which implies both ways.

File: README.md (hunk 3 above)
Comment (ronag) [reply README.md:190]: Isn't this example supposed to show how to do it? Otherwise what's the purpose of the example?

**Review by ronag — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -166,6 +166,33 @@ Only supported on Node 16+.
...
+        const json = await res.json()
+        console.log(json);
+    }
+```
+
+#### `response.body`
+
+Nodejs has two kinds of streams: [web streams](https://nodejs.org/dist/latest-v16.x/docs/api/webstreams.html) which follow the API of the WHATWG web standard found in browsers, and an older Node-specific [streams API](https://nodejs.org/api/stream.html). `response.body` returns a readable web stream. If you would prefer to work with a Node stream you can convert a web stream using `.fromWeb()`. 
+
+```
```
Comment (octet-stream) [inline README.md:185]: Same for this example - add `js`, please.

**Pull request merged**